### PR TITLE
#Issue 335 dependencies removed in the same namespace from HR module

### PR DESCRIPTION
**Pull request by vikasrnv**

### CODE REVIEW

File: app/Models/HR/Application.php [hunk 1]
```diff
@@ -3,12 +3,6 @@
 namespace App\Models\HR;
 
 use App\Events\HR\ApplicationCreated;
-use App\Helpers\ContentHelper;
-use App\Models\HR\Applicant;
-use App\Models\HR\ApplicationMeta;
-use App\Models\HR\ApplicationRound;
-use App\Models\HR\Job;
-use App\User;
```
Comment (rathorevaibhav) [inline app/Models/HR/Application.php]: `App\User` is not in the same namespace. Make sure you test the functionality depending on these files before marking for review.

File: app/Models/HR/Application.php [hunk 2]
```diff
@@ -3,12 +3,6 @@
 namespace App\Models\HR;
 
 use App\Events\HR\ApplicationCreated;
-use App\Helpers\ContentHelper;
```
Comment (rathorevaibhav) [inline app/Models/HR/Application.php]: `App\Helpers\ContentHelper` is not in the same namespace. Make sure you test the functionality depending on these files before marking for review.

File: app/Models/HR/Job.php [hunk 3]
```diff
@@ -2,9 +2,6 @@
 
 namespace App\Models\HR;
 
-use App\Events\HR\JobUpdated;
```
Comment (rathorevaibhav) [inline app/Models/HR/Job.php]: `App\Events\HR\JobUpdated` is not in the same namespace. Make sure you test the functionality depending on these files before marking for review.

**Comment by vikasrnv** [comment]
thanks for your review sir. By retesting namespace dependencies I found that, some dependencies are necessary for the file.. So I Added them again.
please review it again @rathorevaibhav

File: app/Models/HR/Application.php [hunk 4]
```diff
@@ -4,14 +4,9 @@
...
 use App\Helpers\ContentHelper;
-use App\Models\HR\Applicant;
-use App\Models\HR\ApplicationMeta;
-use App\Models\HR\ApplicationRound;
-use App\Models\HR\Job;
-use App\User;
 use Illuminate\Database\Eloquent\Model;
 use Illuminate\Support\Facades\Auth;
-
+use App\user;
```
Comment (rathorevaibhav) [inline app/Models/HR/Application.php]: please check the spelling. it's incorrect.

**Pull request merged**